libs/application/templates/inheritance-report/src/fields/HeirsRepeater/index.tsx

```
@@ -160,11 +160,12 @@ export const HeirsRepeater: FC<
       return
     }
 
-    const total = values.reduce((acc: number, current: any) => {
+    let total = values.reduce((acc: number, current: any) => {
       const val = parseFloat(current[props.sumField])
 
       return current?.enabled ? acc + (isNaN(val) ? 0 : val) : acc
```

Biome (1.9.4)

[error] isNaN is unsafe. It attempts a type coercion. Use Number.isNaN instead.

See the MDN documentation for more details.
Unsafe fix: Use Number.isNaN instead.


(lint/suspicious/noGlobalIsNan)